### PR TITLE
Added the ca_AD ( catalan ) Translation

### PR DESCRIPTION
**Pull request by HardwareBoffin**
## Description:

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [x] The code change is tested and works with Tasmota core ESP32 V.2.0.4.1
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

**Comment by HardwareBoffin** [comment]
Just added the ca_AD.h file and updated the user config to add the commented option.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-![Tasmota logo](/tools/logo/TASMOTA_FullLogo_Vector.svg)
+This is a FORK providing ( in a few days I hope ) the translation file for the Catalan language.
```
Comment (ascillato) [inline README.md:1]: This should not be changed. Please, revert it. Thanks.

**Review by ascillato — CHANGES_REQUESTED** [review]
Please, do not change the README.md file.

**Comment by s-hadinger** [comment]
It's also missing the build definitions in platform_tadmota_env[32].ini

File: README.md (hunk 1 above)
Comment (HardwareBoffin) [reply README.md:1]: Reverted. Apologize for the inconvenience.

**Comment by HardwareBoffin** [comment]
platform_tasmota_env[32].ini modified as requested.

**Comment by Jason2866** [comment]
Where are the changes? Readme still changed and platformioxxx.ini file still missing

**Review by ascillato2 — APPROVED** [review]

**Pull request merged**